app-portage/eclass-manpages/files/eclass-to-manpage.awk

```diff
@@ -28,6 +28,7 @@
 # @FUNCTION: foo
 # @USAGE: <required arguments to foo> [optional arguments to foo]
 # @RETURN: <whatever foo returns>
+# @OUTPUT: <values emitted to STDOUT>
```

Comment:
I'd personally put it above `@RETURN` since output usually happens before returning ;-).